cypress/integration/admin_portal/tests/test_code_management.spec.js

```diff
@@ -0,0 +1,53 @@
+import LandingPage from '../pages/landing_page'
+import CodeManagementPage from '../pages/enterprise_code_management'
+
+describe('landing page tests', function () {
+  const landingPage = new LandingPage()
+  const codeManagementDashboard = new CodeManagementPage()
+
+  before(function(){
+    cy.login_using_api(Cypress.env('ADMIN_PORTAL_USER_EMAIL'), Cypress.env('ADMIN_PORTAL_USER_PASSWORD'))
+  })
+
+  beforeEach(function () {
+    Cypress.Cookies.preserveOnce('edxloggedin', 'stage-edx-user-info', 'stage-edx-sessionid')
+    cy.visit('/')
+    landingPage.goToEnterprise('SuccessFactors')
+    landingPage.openCodeManagement()
```

Comment:
you can skip the above two steps and simple visit the intended page directly

Reply:
When code Management is accessed with the direct URL after the first time, we get 404. So to avoid the situation I had to do it this way.